code/FulpstationCode/fulp_ops.dm

```diff
@@ -0,0 +1,107 @@
+/datum/outfit/syndicate/plasmaman
+	name = "Syndicate Operative Plasmaman - Basic"
+	head = /obj/item/clothing/head/helmet/space/plasmaman/robotics/syndicate
+	uniform = /obj/item/clothing/under/plasmaman/security/syndicate
+	r_hand= /obj/item/tank/internals/plasmaman/belt/full
+	mask = /obj/item/clothing/mask/gas/syndicate
+	gloves = /obj/item/clothing/gloves/combat
+	backpack_contents = list(/obj/item/paper/guides/plasmemeops)
+
+/datum/outfit/syndicate/leader/plasmaman
+	name = "Syndicate Leader Plasmaman - Basic"
+	head = /obj/item/clothing/head/helmet/space/plasmaman/robotics/syndicate
+	uniform = /obj/item/clothing/under/plasmaman/security/syndicate
+	r_pocket= /obj/item/tank/internals/plasmaman/belt/full // They spawn with both hands busy, so they have to manually set internals
```

Comment:
make sure that this won't overwrite anything important

Reply:
the war declare + paper are spawned on hand, which is why i was forced to move internals to r_pocket
(And pretty sure it's empty, left pocket having the makarov and the other items being on backpack)

Reply:
you might want to check that quirks or something can't replace that right pocket slot's item